src/components/textarea/textarea.spec.tsx

```diff
@@ -460,21 +468,19 @@ describe("componentWillUnmount", () => {
             labelAlign="left"
             labelInline
             labelWidth={100}
-            reverse
           />
         </CarbonProvider>
       );
 
     describe("label width and align props", () => {
       it("default to undefined", () => {
         const wrapper = renderWithNewValidations({});
-        const { labelAlign, labelInline, reverse, labelWidth } = wrapper
+        const { labelAlign, labelInline, labelWidth } = wrapper
```

Comment:
I'm probably missing something but it's not clear to me why you've removed the test for `reverse`, since this isn't a prop that's been removed.

Reply:
I'm honestly not sure why this was removed. Can't work out what the reasoning was. Re-added because doesn't seem to be any issues with having it and I'm with you, it's not clear why it was removed in the first place